### ISSUE TITLE
routing table mode didn't work as documented

### ISSUE THREAD

**Issue by bobloblian** [kube-vip/kube-vip#975]
I think this is a documentation issue rather than a bug, hope it's okay to report it here.

I learned about Routing Table mode when setting up kube-vip as a daemonset.  According to that page, enabling it requires only that "both leaderElection and servicesElection flags should not be set".  Following along, removing the --leaderElection flag from the manifest creation command did create a kube-vip.yaml that did not contain the string Election.  However, after deploying the daemonset, I could see in the logs that the leader election was still happening.

Looking closer at the yaml, I discovered that vip_arp was set to true, so I set this to false.  This produced an error:
prematurely exiting Load-balancer as no modes [ARP/BGP/Wireguard] are enabled   

The only other mentions I found searching the docs for more info about Routing Table mode were high-level descriptions, but after a bit of fiddling with the search term on Google, I found that one could explicitly enable Routing Table mode in the yaml by setting vip_routingtable to true; and then in trying to figure out how to do that automatically while creating the manifest, I found in the help that one can add the --table argument while creating the manifest.  

I expect it would benefit those who follow me if the initial paragraph on the daemonsets page explained that ARP must be disabled the arguments to the manifest creation command should have the --leaderElection and --arp removed and the --table added in order to make it work. 

For grins and giggles, I tried setting both --arp and --table, which resulted in the Mode being set to Routing Table, but elections still being held.  I kinda think that should produce an error?  Maybe there is a reason they can are not mutually exclusive? 

I also tried setting arp and vip_leaderElection to false in the yaml with vip_routingtable as true, and Routing Table mode works with that, so perhaps that paragraph could also be updated to note that the leader/serviceElection must be not set or set to false.

**To Reproduce**
As above

**Expected behavior**
As above

**Environment (please complete the following information):**
 - OS/Distro:
cat /etc/debian_version 
trixie/sid
 - Kubernetes Version: [e.g. v.1.18]
1.31
 - Kube-vip Version: [e.g. 0.2.3]
app.kubernetes.io/version: v0.8.4

**Comment by Cellebyte** [comment]
@bobloblian You are right the documentation around routing table mode is only high level.

You can still run routing table mode with leader election.
Leader election just ensures that only a single path is advertised for the specific service and one node takes the lead for the specific service. 

arp mode just binds the IP to a specific interface. And starts an arp socket for the IP on it. This should not be needed if you want to use routing. But also does not harm.
